### PR TITLE
Removed duplicate "bouwdossier" + Removed queryFormatter and queryMat…

### PR DESCRIPTION
**Pull request by chrisvanmook**
…cher as they aren't used anymore.

### CODE REVIEW

File: src/graphql/search/data/index.ts [hunk 1]
```diff
@@ -58,45 +54,40 @@ const index = async (
...
+      // The Promise resolved, so return the valid Data
+      const { results = [], count } = result.value || {}
+
+      return {
+        count: count || 0,
+        label: count === 1 ? labelSingular : label,
+        type,
+        status: result.status,
+        results:
+          results.length > 0 && limit // TODO: Add test to see if the correct number of results is returned
```
Comment (janjaap) [inline src/graphql/search/data/index.ts:86]: Why not add that test to this PR?

Comment (chrisvanmook) [reply src/graphql/search/data/index.ts:86]: A bit out of scope for this ticket (didn't add the todo), but will create a test for it

File: src/graphql/search/data/config.ts [hunk 2]
```diff
@@ -122,16 +120,6 @@ export const DATA_SEARCH_ENDPOINTS: Array<DataSearchType> = [
...
     label: 'Bouw- en omgevingsdossiers',
     searchParam: 'dossier',
-    queryMatcher: /^[A-Z]{2,3}\d/,
-  },
-  {
-    endpoint: SEARCH_ENDPOINTS[DataType.ConstructionFiles],
-    type: DataType.ConstructionFiles,
-    labelSingular: 'Bouw- en omgevingsdossier',
-    label: 'Bouw- en omgevingsdossiers',
-    searchParam: 'olo_liaan_nummer',
```
Comment (jonkoops) [inline src/graphql/search/data/config.ts:132]: This seems to match a different parameter, that's not an issue?

**Review by jonkoops — APPROVED** [review]

File: src/graphql/search/data/config.ts (hunk 2 above)
Comment (chrisvanmook) [reply src/graphql/search/data/config.ts:132]: Just found out the `searchParam` doesn't do anything, so I've made it optional

**Pull request closed without merging**